EMQX/EMQX.php

```diff
@@ -0,0 +1,35 @@
+<?php namespace App\SupportedApps\EMQX;
+
+class EMQX extends \App\SupportedApps implements \App\EnhancedApps {
+
+    public $config;
+
+    //protected $login_first = true; // Uncomment if api requests need to be authed first
+    //protected $method = 'POST';  // Uncomment if requests to the API should be set by POST
+
+    function __construct() {
+        //$this->jar = new \GuzzleHttp\Cookie\CookieJar; // Uncomment if cookies need to be set
+    }
+
+    public function test()
+    {
+        $test = parent::appTest($this->url('status'));
+        echo $test->status;
+    }
+
+    public function livestats()
+    {
+        $status = 'inactive';
+        $res = parent::execute($this->url('status'));
+        $details = json_decode($res->getBody());
+
+        $data = [];
```

Comment:
Still work in progress?

Reply:
No clue, just requested it from the heimdall site (https://apps.heimdall.site/request/list), following the request manual ...

Reply:
OK, I see. Somebody would need to also write the code for this, it's just the template so far.